health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/BannerCard.js

```
@@ -0,0 +1,152 @@
+  
+  const getSubHeading = () => {
+    const date = new Date();
+    // const zeroTime = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+
+    const formattedStartTime = format(requestDate?.startDate, "dd-MMM-yyyy HH:mm:ss");
+    const formattedEndTime = format(requestDate?.endDate, "dd-MMM-yyyy HH:mm:ss");
+    return `${t("SUB_" + chartName)} ${formattedStartTime} to ${formattedEndTime}`;
+  };
+
+  const chartData = useMemo(() => {
+    let data = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)